apps/govuk-docs/check-links.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash +x
```

Comment:
Do we still need the `+x`?

Reply:
Is it better to do `#! /bin/env bash` these days?

Reply:
Good question! Not really a short answer, but I think it boils down to using `/bin/env` is more portable but _slightly_ less secure. I don't think either is a strong enough consideration to influence this context though.